docs/server-docs-release-process.adoc

```diff
@@ -0,0 +1,20 @@
+= Server docs release process
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+
+== List of jobs to complete before releasing
+
+* Update version under the `server_version` key in `jekyll/_config.yml`.
+* Make operational changes required for the release.
+* If anyth
```

Comment:
What is this?

Reply:
😭good question, will fix! - this is mainly a list for me so I don't miss anything. Must have got distracted :-O.